docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.fonts.scss

```diff
@@ -0,0 +1,126 @@
+// Started using humsci_basic `hb` prefix here
+// Thoughts?
+@mixin hb_heading-1 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 40px;
+    line-height: 126%;
+
+    @include grid-media-min("md") {
+      font-size: 58px;
+    }
+  }
+}
+
+@mixin hb_heading-2 {
+  @include colorful {
+    font-weight: theme-font-weight(semibold);
+    font-size: 33px;
+    line-height: 125%;
```

Comment:
I don't think I've ever used a percentage for line heights. Have we tested how these will scale when zooming into the page? Whenever I see a percentage it's usually based on the width of the parent element, will element widths affect the line-heights of different elements?

Reply:
we learned from MDN that it's acceptable and works! https://developer.mozilla.org/en-US/docs/Web/CSS/line-height but we could also always substitute, for example, 126% for 1.26 for line-height